src/main/java/org/capstone/job_fair/constants/DataConstraint.java

```diff
@@ -132,6 +132,8 @@ private JobPosition() {
         public static final int SKILL_TAG_MIN = 1;
         public static final int EMPLOYEE_MIN = 1;
         public static final int EMPLOYEE_MAX = 100000;
+        public static final int OFFSET_MIN = 0;
```

Comment:
Move to different constant class please, cause the default paging offset will be the same for many endpoint